README.md

```
@@ -132,49 +101,69 @@ bcftools annotate \
+  --vcf --cache --refseq --exclude_predicted --hgvs --symbol --check_existing \
+  --canonical --mane --numbers --pick --format vcf --offline --exclude_null_alleles \
+  --assembly GRCh38 \
+  --fields SYMBOL,Consequence,Feature,HGVSc,HGVSp,EXON,INTRON,CDS_position,Protein_position,CANONICAL,MANE,MANE_SELECT,MANE_PLUS_CLINICAL \
+  --shift_3prime 1 \
+  --buffer_size 500 \
+  --no_stats \
+  --compress_output bgzip
 ```
-
+Split out VEP annotations to separate INFO fields. prefixed with `_CSQ`:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)